content/docs/tasks/traffic-management/egress/egress-kubernetes-services/index.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Kubernetes Services for Egress Traffic
+description: Shows how to configure Istio  Kubernetes External Services.
+keywords: [traffic-management,egress]
+weight: 60
+---
+
+Sometimes, for various reasons, for example in pre-Istio scenarios,
+Kubernetes [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
+services or Kubernetes services with
+[Endpoints](https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors) are used to
+access external services.
+This task shows that these Kubernetes mechanisms for accessing external services continue to work with Istio.
+The only configuration step you must perform is to disable Istio's
+[mutual TLS](/docs/concepts/security/#mutual-tls-authentication).
```

Comment:
Requiring folks to disable mTLS to get common K8s features is not the right approach, IMO. @geeknoid WDYT?
At the very least, readers will want to know why that is the case.

Reply:
@rcaballeromx You are right, other TLS mode, for example SIMPLE, should be used instead of Istio mutual TLS.

Reply:
I agree with @rcaballeromx, you should add a couple of sentences to explain what is needed and why.